tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -132,7 +132,14 @@ object Discover {
 
   /** Versions of clang which are known to work with Scala Native. */
   private[scalanative] val clangVersions =
-    Seq(("6", "0"), ("5", "0"), ("4", "0"), ("3", "9"), ("3", "8"), ("3", "7"))
+    Seq(("8", ""),
+        ("7", ""),
```

Comment:
`("8", "0")` and `("7", "0")`.

Reply:
Done ("8", "0") and ("7", "0") added to previous variant with empty string second member of tuple.
Belts & suspenders engineering which does not affect binary size of end user images.

Re-tested with clang-7 (test-all) and passed.  

Clang-8 just went to rc5. I am glad they are taking the time to remove some nasty bugs
but that makes it slide another week. I do not foresee problems with clang-8 but will
install it and run test-all in both debug & release modes.

Reply:
Yes, the clangVersions data structure is getting large, particularly now that there are two entries per
clang/llvm version.   I tried to be respectful of prior art.  

Do you know the Swiss child psychologist Jean Piaget's accommodation/assimilation model of
child development?  Basically he describes a situation where changes to a child's mental
schema pile up (accommodation) until the complexities force a re-organization (assimilation) or a
simpler schema is found.

If I were writing this section from a blank sheet of paper, I would probably just use the default `clang`
verb. Much as I hate environmental variables, I would probably have a environment variables to
set LLVM version and set clang, clang++, and anything else off of it, superceeding anything set up
by llvm-config.  There are also a number of ways to set up alternatives in the OS.  

The existing 'add a version to clangVersions' works and fits available time, mine for development
and yours for review, but  it should probably be changed for clang-9 or clang-10.  Other
fish to fry (well, broil) now.